### PR TITLE
Media search result enhancement

### LINKED ISSUE #1737: Media search result enhancement
**Is your feature request related to a problem? Please describe.**
The media search result needs enhancements.

**Describe the solution you'd like**
1. Result with no image should have a placeholder: https://prnt.sc/vd7yt2
2. Load thumbnail version of images if original size is not needed: https://prnt.sc/vd7rhe 
3. Responsive layout can be improved: https://prnt.sc/vd7rhe

### PR DESCRIPTION
**Pull request by amincdev**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #1737 

### How to test the changes in this Pull Request:

1. Check result with no image should have a placeholder now
2. Loading thumbnail version of images now
3. Responsive layout improved

### Proof Screenshots or Video
https://prnt.sc/vgpzhm
https://prnt.sc/vgpd1p

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Comment by amincdev** [comment]
@chetansatasiya Can you please review this?

File: src/bp-templates/bp-nouveau/buddypress/search/loop/albums-ajax.php [hunk 1]
```diff
@@ -18,6 +18,10 @@
 					<a href="<?php echo esc_url( $albums_link ); ?>">
 						<img src="<?php echo esc_url( $media_album_template->album->media['medias'][0]->attachment_data->thumb ); ?>" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
 					</a>
+				<?php else: ?>
+					<a href="<?php echo esc_url( $albums_link ); ?>">
+						<img src="<?php echo buddypress()->plugin_url; ?>bp-templates/bp-nouveau/images/placeholder.png" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/albums-ajax.php:23]:
```suggestion
						<img src="<?php echo esc_url( buddypress()->plugin_url ); ?>bp-templates/bp-nouveau/images/placeholder.png" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
```

File: src/bp-templates/bp-nouveau/buddypress/search/loop/albums.php [hunk 2]
```diff
@@ -15,9 +15,13 @@
...
 					<?php if ( ! empty( $media_album_template->album->media['medias'] ) ) : ?>
-					<a href="<?php echo esc_url( $albums_link ); ?>">
-						<img src="<?php echo esc_url( $media_album_template->album->media['medias'][0]->attachment_data->thumb ); ?>" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
-					</a>
+						<a href="<?php echo esc_url( $albums_link ); ?>">
+							<img src="<?php echo esc_url( $media_album_template->album->media['medias'][0]->attachment_data->thumb ); ?>" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
+						</a>
+					<?php else: ?>
+						<a href="<?php echo esc_url( $albums_link ); ?>">
+							<img src="<?php echo buddypress()->plugin_url; ?>bp-templates/bp-nouveau/images/placeholder.png" alt="<?php echo wp_kses_post( bp_get_album_title() ); ?>" />
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/search/loop/albums.php:23]: Same here needs escaping.

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@amincdev Do the changes as requested and fix the conflict and then merge in dev.

File: src/bp-templates/bp-nouveau/buddypress/search/loop/albums-ajax.php (hunk 1 above)
Comment (amincdev) [reply src/bp-templates/bp-nouveau/buddypress/search/loop/albums-ajax.php]: @chetansatasiya  Changes added

File: src/bp-templates/bp-nouveau/buddypress/search/loop/albums.php (hunk 2 above)
Comment (amincdev) [reply src/bp-templates/bp-nouveau/buddypress/search/loop/albums.php]: @chetansatasiya Changes added

**Review by chetansatasiya — APPROVED** [review]
@amincdev Approved!!!

**Comment by chetansatasiya** [comment]
@amincdev Merge in dev and add to testing.

**Comment by amincdev** [comment]
@ankitguptaindia Merged in `dev` and ready for testing.

**Comment by sumitbuddyboss** [comment]
Hello @amincdev 

Fixed > I've checked in dev branch and the Below points are working as expected, For proof please check the video.
1) Check the result with no image should have a placeholder now
2) Loading thumbnail version of images now
3) Responsive layout improved

Video:
https://www.loom.com/share/f213f5acde354f9dbe7d6ebe75bb4fe5

@chetansatasiya 
Ready for release 🚀 

CC @ankitguptaindia

**Pull request merged**